eth/downloader/queue.go

```diff
@@ -803,11 +802,6 @@ func (q *queue) DeliverBodies(id string, txLists [][]*types.Transaction, uncleLi
 		if types.DeriveSha(types.Transactions(etxLists[index]), trieHasher) != header.EtxHash() {
 			return errInvalidBody
 		}
-		if nodeCtx < common.ZONE_CTX {
```

Comment:
Please just create an issue to un-neuter

Reply:
https://github.com/dominant-strategies/go-quai/issues/545